canvas_account_admin_tools/templates/canvas_account_admin_tools/dashboard_account.html

```diff
@@ -4,68 +4,75 @@
 
 {% block stylesheets %}
 <link href="{% static 'canvas_account_admin_tools/css/dashboard.css' %}" rel="stylesheet" />
+  <style>
+    a.btn {
+        width: 20em;
+        margin-top: 1em;
+    }
+  </style>
 {% endblock stylesheets %}
 
 {% block content %}
-<header>
-    <div class="row lti-header">
-        <h1>Administration Tasks</h1>
-    </div>
-</header>
 
-<main>
+    <header>
+      <nav class="ng-scope">
```

Comment:
Why does the `nav` element need to have the scope attached manually?  That's usually something that angular does to elements that need it.